sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_compiler_opencl.asciidoc

```diff
@@ -115,6 +241,11 @@ creating an OpenCL library.
 The kernel compiler can be used to create an OpenCL program, but not an OpenCL
 library.
 
+The `-cl-std=` option is required when compiling kernels that use OpenCL C 2.0
+or OpenCL C 3.0 features.
```

Comment:
```suggestion
The `-cl-std=` option is required when compiling kernels that use OpenCL C 2.0
or OpenCL C 3.0 features. Otherwise, an implementation will default to the highest
minor version of OpenCL 1 supported.
```

I had to go and check the OpenCL specification to figure out why OpenCL 1.1 and 1.2 weren't in this list.

I think we should either not say anything at all (i.e., remove the note about OpenCL 2.0 and OpenCL 3.0, and just say something like "The `-cl-std=` option can be used to control the OpenCL language features available, see Section 5.8.6.5") or give a brief summary of all the options as proposed above.

I don't have a strong preference either way.

Reply:
OK.  I added the clarifying text in 15193cff1dca915f37f02b59ee9bbe7a5f272e57.